util/exec.go

```diff
@@ -62,6 +63,32 @@ func CommandWithResult(ctx context.Context, logger *log.Logger, command *Command
 	return writer.String(), nil
 }
 
+// Flags returns a list of flags bound on the command or nil.
+func Flags(cmd any) (args []string) {
+	typ := reflect.TypeOf(cmd)
+
+	// Dereference pointer
+	if typ.Kind() == reflect.Pointer {
+		typ = typ.Elem()
+	}
+
+	if typ.Kind() != reflect.Struct {
+		return nil
+	}
+
+	for i := 0; i < typ.NumField(); i++ {
+		field := typ.Field(i)
+		arg, ok := field.Tag.Lookup("long")
```

Comment:
Because currently we do not use "short". With the addition of "short" the whole flag parsing also needs to be changed as it currently requires the "long" variant with "--".
Could do but the question is if it is something we should do now or when we need it.